server/tests/users.test.ts

```diff
@@ -515,10 +514,10 @@ describe(`DELETE ${TEST_URLS.usersMe}`, () => {
         beforeAll(async () => {
             const response = await request(BASE_URL).post(TEST_URLS.newUser).send(TEST_USER)
             expect(response.statusCode).toBe(201)
-            token = response.body.data
+            token = response.body.data.token
         })
     
-        it('Should return 200', async () => {
+        it('Should return 201', async () => {
```

Comment:
If it is 201 why are you evaluating 200 in line 522?

Reply:
It was a mistake while following the error, I'll reset the text

Reply:
👌🏻